lib/eval_test.js

```diff
@@ -116,6 +116,7 @@ function composeNormalizer(normalizers) {
  * status of this test (whether it passed, failed, is a placeholder, etc.)
  */
 function evalTest( testCase, apiResults, context ){
+  context = _.clone(context);
```

Comment:
Good catch. I didn't know this was a parameter because only lodash has it, not underscore. interesting